cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -1024,7 +1029,8 @@ def to_pandas(  # type: ignore [override]
         prop_df = local_import("pandas").json_normalize(df.pop("properties"), max_level=2)
         if remove_property_prefix and not prop_df.empty:
             # We only do/allow this if we have a single source:
-            view_id, *extra = set(vid for item in self for vid in item.properties)
+            typed_view_ids = {item.get_source() for item in self if hasattr(item, "get_source")}
```

Comment:
Should this use `isinstance(self, (TypedEdge, TypedNode))` instead?

Reply:
It was just a cheap way to avoid local imports.